lib/pull.js

```diff
@@ -334,11 +351,19 @@ async function downloadAndSave(sourceInformation, token) {
   }
 }
 
-function pull() {
+/**
+ * @param {{ meta: Object.<string, string> }} options
+ */
+function pull(options) {
+  const { meta } = options;
   const token = config.getToken(consts.CONFIG_FILE, consts.API_HOST);
   const sourceInformation = config.parseSourceInformation();
 
-  return downloadAndSave(sourceInformation, token);
+  if (meta?.context === 'github-action') {
```

Comment:
In the near future, I want to.. 
- convert this entire repository to TypeScript
- pin the minimum required Node version

Until then though, I think we should avoid using optional chaining (and other features not introduced till later versions) since this could technically cause breakage for anyone using our CLI with an older version of Node.

Let me know if you disagree!